scripts/zones/Oldton_Movalpolos/mobs/Bugallug.lua

```diff
@@ -14,6 +14,17 @@ entity.onMobEngaged = function(mob)
 end
 
 entity.onMobDeath = function(mob, player, isKiller)
+    local party = player:getParty()
+
+    for _, v in ipairs(party) do
+        if
+            player:getZone() == v:getZone() and
```

Comment:
quest lua

Reply:
Fixed in: https://github.com/AirSkyBoat/AirSkyBoat/pull/397/files